addons/website/static/src/js/editor/widget_link.js

```diff
@@ -73,6 +107,30 @@ weWidgets.LinkTools.include({
         }
         $selectMenu.data("anchor-for", urlInputValue);
     },
+    /**
+     * Update URL according to chosen action
+     *
+     * @private
+     */
+    _adaptURLToAction: function (url) {
+        if (this.$link[0].dataset.linkAction !== 'download') {
+            url = url.replace(/[&?]+download=true/, '');
+        } else if (!/[&?]+download=true/.test(url)) {
+            url = url + (url.includes('?') ? '&download=true' : '?download=true');
+        }
+        return url;
+    },
+    /**
+     * Reset action to default value ie. Open in a new tab
+     *
+     * @private
+     */
+    _resetLinkAction: function () {
+        this.$('.o_we_link_action_change .active').removeClass('active');
+        this.$('.o_we_link_action_change we-button:contains("Open in a new tab")').addClass('active');
+        this.$('.o_we_link_action_change we-toggler').text(_t("Open in a new tab"));
```

Comment:
One selector searching for an untranslated string followed by that translated string... Anyways, we simply should not search items by their text content. Also, that translated string should simply not be hardcoded here: it already is in the tempalte, just retrieve it here.